exercises/java/src/main/java/com/adaptionsoft/games/categories/Pop.java

```diff
@@ -0,0 +1,27 @@
+package com.adaptionsoft.games.categories;
```

Comment:
Interesting that you've split the categories using inheritance, do you think you've introduced any duplication by doing so?  Are there any alternative strategies you could have used?